ocs_ci/templates/logging-deployment/elasticsearch_operator/eo-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-operators-redhat
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-operators-redhat
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+namespace: openshift-operators-redhat
```

Comment:
missing spaces before `namespace`. And, I don't think we have `prometheus-k8s` ServiceAccount  in  this namespace. It is in `openshift-monitoring` namespace unless you have your own prometheus instance.

Reply:
@umangachapagain I am following this openshift doc https://docs.openshift.com/container-platform/4.1/logging/efk-logging-deploying.html. In the eo-rbac.yaml file promotheus-k8s serviceaccount was mentioned in the namespace openshift-opeators-redhat.

Reply:
space issue fixed